pages/service-usages/basic-usages.md

```diff
@@ -1,6 +1,6 @@
 ## Basic Usages
 
-[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/522bf43d2b5f9f09f7ee)
+<a href="https://app.getpostman.com/run-collection/522bf43d2b5f9f09f7ee"> <button class="postman">Run in Postman</button> </a>
```

Comment:
Could you check if assigning the class to the link would have the same effect? you could use class btn for the same block feature

Reply:
On the link itself
![image](https://user-images.githubusercontent.com/41168603/119482037-13b1ea80-bd5c-11eb-958a-cd9254fcdd2c.png)

Reply:
since the run in postman is a link lets remove the button so we maintain the correct semantics.

Reply:
Done.